src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -175,12 +177,26 @@ const asyncVirusScanning = (
 >[] => {
   return responses.map((response) => {
     if (isQuarantinedAttachmentResponse(response)) {
+      if (growthbook) {
+        // trigger guardduty scan
+        SubmissionService.triggerGuarddutyScanThenDownloadCleanFileChain(
+          response,
+          formId,
+        ).mapErr((err) =>
+          logger.info({
+            message: `GuardDuty Scan unsuccessful:, ${err}`,
+            meta: {
+              action: 'Testing guardduty error',
```

Comment:
/nit we usually try to relate the `action` tag back to the function that's being called so that we can easily search for it in code i.e., `handleGetEncryptedResponse`

```suggestion
              action: 'triggerGuarddutyScanThenDownloadCleanFileChain',
```

Reply:
Updated this! Thanks for the suggestion